doc/whats_new.rst

```diff
@@ -107,10 +113,8 @@ Changes
   argument to control whether the colormap is centered around 0 and a `vmin`
   argument.
 
-- :func:`nilearn.plotting.view_img_on_surf`, :func:`nilearn.plotting.view_surf`
-  and :func:`nilearn.plotting.view_connectome` now allow disabling the colorbar,
-  and setting its height and the fontsize of its ticklabels.
-
+- Users can now control the size and fontsize of colorbars in interactive
+  surface and connectome plots, or disable the colorbar alltogether.
```

Comment:
Spelling: altogether.
Disable and altogether are redundant. I would lwrite "disable the colorbar." and be done.